### ISSUE TITLE
[documentation] "Quick Start" needs an actual Quick start

### ISSUE THREAD

**Issue by Redsandro** [intel-analytics/ipex-llm#11521]
I am super excited to try the new Intel Core Ultra + Arc iGPU with AI applications, and LLMs in particular.

I'm a long time Python dev but Intel (for development) is new to me. In particular, documentation by Intel. So in my frame of reference, whenever I read a Quick Start by Intel, for example [this one for the WebuU](https://github.com/intel-analytics/ipex-llm/blob/main/docs/mddocs/Quickstart/webui_quickstart.md) but it's a recurring theme, it's actually a long document with many redirects to other lengthy "Quick Starts" with many options and things to choose from, and 9 out of 10 times no Linux-specific information.

So I spend an hour reading and solving errors from suggested command lines, and after resolving everything I'm left with the final error message: _Ubuntu 24.04 is not supported for this package._ (Even though it had 24 in the name, confusing me.)

These are not "Quick Start"s, but perhaps "Start Here" or "Installation" documents in my opinion. I've built hardware accelerated model implementations for nVidia hardware on Linux that had a Quick Start that fits on one page. Now if you want something crazy, like a pineapple, you'll need to click on some advanced links. But shouldn't the Quick Start be quicker to qualify for the title?

I won't be downgrading my Linux version to try this brand new Intel Core Ultra + Arc iGPU, but if I can redirect the hardware I'll gladly try again in Docker. Although if that would work, I guess there would already be "docker pull" commands in the "Quick Start" guides.

**Comment by sgwhat** [comment]
Thank you for using IPEX-LLM. We will check the support for Ubuntu 24.04 and update the corresponding document.

**Comment by tristan-k** [comment]
I'm in the same boat. I recently bought a NUC14 with a Intel Arc Graphics (iGPU 7Xe/112EU/896SP, Xe-LPG / Gen 12.7) but I'm unable to use Intel OneAPI because it only supports Ubuntu 22.04. At the same time I will have to use at least [Kernel 6.7](https://www.reddit.com/r/IntelArc/comments/1b8ku34/comment/ktrf0dk/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) because installing the [iGPU driver modules on Kernel 6.5](https://github.com/intel-analytics/ipex-llm/blob/main/docs/mddocs/Quickstart/install_linux_gpu.md#for-linux-kernel-65) with Ubuntu 22.04 will freeze the whole system. 

In conclusion: 

- I cant use Ubuntu 22.04 with Kernel 6.5 and `intel-i915-dkms` because the system will freeze
- I cant use Ubuntu 24.04 with Kernel 6.8 because Intel OneAPI only supports Ubuntu 22.04

**Comment by jason-dai** [comment]
> I'm in the same boat. I recently bought a NUC14 with a Intel Arc Graphics (iGPU 7Xe/112EU/896SP, Xe-LPG / Gen 12.7) but I'm unable to use Intel OneAPI because it only supports Ubuntu 22.04. At the same time I will have to use at least [Kernel 6.7](https://www.reddit.com/r/IntelArc/comments/1b8ku34/comment/ktrf0dk/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) because installing the [iGPU driver modules on Kernel 6.5](https://github.com/intel-analytics/ipex-llm/blob/main/docs/mddocs/Quickstart/install_linux_gpu.md?rgh-link-date=2024-07-13T17%3A56%3A17Z#for-linux-kernel-65) with Ubuntu 22.04 will freeze the whole system.
> 
> In conclusion:
> 
> * I cant use Ubuntu 22.04 with Kernel 6.5 and `intel-i915-dkms` because the system will freeze
> * I cant use Ubuntu 24.04 with Kernel 6.8 because Intel OneAPI only supports Ubuntu 22.04

Does this help? https://github.com/intel-analytics/ipex-llm/issues/11568#issuecomment-2227157685

**Comment by tristan-k** [comment]
@jason-dai 

No. There is something serious wrong with the driver. I managed to ssh into the machine after enabling `sudo systemctl set-default multi-user.target` but I'm unable to load anything related to the gpu. I can however run `xpu-smi`. For example I tried to run [vkpeak](https://github.com/nihui/vkpeak) but the applications just freezes. If I then try to reboot the whole system locks up and I will have to do a hard reset.

```
$ sudo xpu-smi discovery -d 0
+-----------+--------------------------------------------------------------------------------------+
| Device ID | Device Information                                                                   |
+-----------+--------------------------------------------------------------------------------------+
| 0         | Device Type: GPU                                                                     |
|           | Device Name: Intel(R) Arc(TM) Graphics                                               |
|           | PCI Device ID: 0x7d55                                                                |
|           | Vendor Name: Intel(R) Corporation                                                    |
|           | SOC UUID: 00000000-0000-0200-0000-00087d558086                                       |
|           | Serial Number: unknown                                                               |
|           | Core Clock Rate: 2200 MHz                                                            |
|           | Stepping: C0                                                                         |
|           | SKU Type: N/A                                                                        |
|           |                                                                                      |
|           | Driver Version: I915_24.3.23_PSB_240419.26                                           |
|           | Kernel Version: 6.5.0-41-generic                                                     |
|           | GFX Firmware Name: GFX                                                               |
|           | GFX Firmware Version: unknown                                                        |
|           | GFX Firmware Status: unknown                                                         |
|           |                                                                                      |
|           | PCI BDF Address: 0000:00:02.0                                                        |
|           | PCI Slot: N/A                                                                        |
|           | PCIe Generation: -1                                                                  |
|           | PCIe Max Link Width: -1                                                              |
|           |                                                                                      |
|           | Memory Physical Size: 0.00 MiB                                                       |
|           | Max Mem Alloc Size: 4095.99 MiB                                                      |
|           | ECC State: N/A                                                                       |
|           | Number of Memory Channels: N/A                                                       |
|           | Memory Bus Width: N/A                                                                |
|           | Max Hardware Contexts: 65536                                                         |
|           | Max Command Queue Priority: 0                                                        |
|           |                                                                                      |
|           | Number of EUs: 112                                                                   |
|           | Number of Tiles: 1                                                                   |
|           | Number of Slices: 1                                                                  |
|           | Number of Sub Slices per Slice: 7                                                    |
|           | Number of Threads per EU: 8                                                          |
|           | Physical EU SIMD Width: 8                                                            |
|           | Number of Media Engines: 2                                                           |
|           | Number of Media Enhancement Engines: 1                                               |
|           |                                                                                      |
|           | Number of Xe Link ports: N/A                                                         |
|           | Max Tx/Rx Speed per Xe Link port: N/A                                                |
|           | Number of Lanes per Xe Link port: N/A                                                |
+-----------+--------------------------------------------------------------------------------------+
```
